napari/layers/transforms.py

```diff
@@ -24,6 +24,8 @@ def __init__(self, func=tz.identity, inverse=None, name=None):
         self.name = name
         if func is tz.identity:
             self._inverse_func = tz.identity
+        self.translate = None
+        self.scale = None
 
```

Comment:
This means we don't run into AttributeErrors later on, and keeps the downstream code a little more elegant.